lib/core/utils/preload-cssom.js

```diff
@@ -75,14 +149,24 @@ function loadCssom({ root, shadowId }, timeout, convertTextToStylesheetFn) {
 	});
 
 	// iterate to decipher multi-level nested sheets if any (this is essential to retrieve styles from shadowDOM)
-	sheets.forEach(sheet => {
+	sheets.forEach((sheet, sheetIndex) => {
+		/* eslint max-statements: ["error", 20] */
+
+		// basic priority
+		const sheetLevelPriority = `${rootIndex}${sheetIndex}`;
```

Comment:
I don't think I understand what's happening here. It looks like we're concatenating (not adding) numbers together in a string to be used as a numerical value later on. Can you explain how this works?

I'm stuck on the following:

A "priority" string is created with a `rootIndex` of `1` and a `sheetIndex` of `3000`. It is then coerced to the integer `13000`.

Later on, another string is created with `rootIndex` of `2` and `sheetIndex` of `1`. It's then coerced to `21`.

`13000` is larger than `21`, but its "root" was higher. Is this OK?